src/index.css

```diff
@@ -417,48 +460,38 @@ footer a {
 #playButton {
     font-family: var(--battle-font);
     font-weight: 1000;
-    font-size: 1.5rem;
+    font-size: 2rem;
     letter-spacing: 0.35rem;
-    line-height: 2.25rem;
-}
-
-.modal-title {
-    color: var(--modal-title);
-    font-weight: 1000;
-    text-shadow: 5px 0 var(--modal-text-shadow);
+    line-height: 2rem;
 }
 
 .welcome {
-    background-color: var(--modal-text-background);
-    border: 10px ridge var(--modal-text);
-    border-radius: 0.25rem;
+    border-radius: 0.5rem;
     padding: 0.5rem;
     width: 90%;
-    text-align: center;
+}
+
+.welcome p {
+    text-transform: uppercase;
+    color: var(--off-white);
+    padding: 1rem 3rem;
+    margin: 0;
 }
 
 #playButton {
-    color: var(--modal-play-button-text);
-    text-shadow: 1px 1px 3px var(--modal-text-shadow);
+    color: var(--quinary);
+    text-shadow: 1px 1px 3px var(--main-yellow);
     background-color: transparent;
     border: none;
     text-transform: uppercase;
     cursor: pointer;
-    padding-bottom: 1rem;
+    padding: 0 0 1rem 3rem;
 }
 
 #playButton:enabled:hover {
```

Comment:
Could we add a focus state to this as well? I think currently, the button does not highlight on tab focus. Other than that, everything looks awesome:)

Reply:
Good catch! Done!